tests/src/test/scala/cats/mtl/tests/SummonableImplicits.scala

```diff
@@ -15,27 +15,24 @@ final class SummonableImplicits extends BaseSuite {
       def _1 = ApplicativeAsk[ReaderStrId, String]
       def _2 = ApplicativeAsk[ReaderStrInt, Int]
       def _3 = ApplicativeAsk[ReaderStrInt, String]
-      def _4 = ApplicativeAsk[ReaderStrFuncInt, Int]
     }
 
     val FunctorListenTest: Unit = {
       def _1 = FunctorListen[WriterStrId, String]
       def _2 = FunctorListen[WriterTStrWriterTInt, String]
       def _3 = FunctorListen[WriterTStrWriterTInt, Vector[Int]]
-      def _4 = FunctorListen[WriterTStrTupleInt, Vector[Int]]
     }
 
     val FunctorRaiseTest: Unit = {
       def _1 = FunctorRaise[EitherStrId, String]
-      def _2 = FunctorRaise[EitherTStrEitherTInt, Int]
+      //def _2 = FunctorRaise[EitherTStrEitherTInt, Int]
```

Comment:
This is the only one I couldn't get to infer correctly, it should be picked up by nature of `functorRaiseForMonadPartialOrder` but for some reason it doesn't.

Reply:
Glancing at the structure, I would *guess* it runs afoul of the same bug that hits shims and forces the use of a macro replacing implicit search (namely, recursive falsification). Hard to say without looking closer though